org.ppsspp.PPSSPP.yml

```diff
@@ -51,3 +55,26 @@ modules:
 
       - type: file
         path: org.ppsspp.PPSSPP.metainfo.xml
+
+  - name: ppsspp-localization
+    buildsystem: simple
+    build-commands:
+      - |
+        for LANG_FILE in assets/lang/*.ini
+          do ln -fsr "$FLATPAK_DEST/share/locale/$(basename "$LANG_FILE" | grep -Eo '^[a-z]{2}')/ppsspp/$(basename "$LANG_FILE")" "$FLATPAK_DEST/share/ppsspp/$LANG_FILE"
```

Comment:
Better to use var.  
```
for LANG_FILE in assets/lang/*.ini
    LANG_NAME=$(basename "$LANG_FILE");
    LANG_DEST="$FLATPAK_DEST/share/locale/${LANG_NAME:0:2}/ppsspp/$LANG_NAME";
    ln -fsr "$LANG_DEST" "$FLATPAK_DEST/share/ppsspp/$LANG_FILE";
    install -Dm644 "$LANG_FILE" "$LANG_DEST";
done
```

Reply:
Yeah, I thought so too. On the other hand I did not want to pollute the environment variable space unnecessarily with even more variables. Additionally, this script loop has a very limited number of iterations and basically runs only once on some machine in the cloud during build time. So, no biggie, it is good enough.

Reply:
I mean that if there is no var, I can't figure out what the code is doing for the first time.  
It's a bit confusing of `ln -fsr "$FLATPAK_DEST/share/locale/$(basename "$LANG_FILE" | grep -Eo '^[a-z]{2}')/ppsspp/$(basename "$LANG_FILE")" "$FLATPAK_DEST/share/ppsspp/$LANG_FILE"`.